assets/css/card.scss

```diff
@@ -0,0 +1,116 @@
+.dx-card {
+    display: flex;
+    position: relative;
+    padding: 2.25rem 1.5rem 1.5rem;
+    margin: 2.5rem 0 1rem;
+    flex-direction: column;
+    border-radius: 4px;
+    box-shadow: 1px 2px 10px rgba(0, 0, 0, 0.1);
+    color: $brown;
+    text-decoration: none;
+
+    &:hover,
+    &:focus {
+        box-shadow: 1px 2px 20px rgba(0, 0, 0, 0.15);
+        color: $brown;
+
+        &:after {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5681062)